control/ipmitool.spec.in

```diff
@@ -359,7 +983,7 @@ fi
  - Fix SEL event decoding for generic events
  - Handle empty SEL gracefully when doing "sel list"
  - Fix sdr handling of sensors that do not return a reading
- - Fix for CSV display of sensor readings/units from Fredrik �hrn
+ - Fix for CSV display of sensor readings/units from Fredrik ohrn
```

Comment:
I don't think so. You should be using a UTF-8 compatible locale and editor, because that was `Fredrik Öhrn` and should stay so.

Reply:
OK, fixed. I was using UTF8 but something went wrong when I imported the text from the Changelog file.